nbsite/nbbuild.py

```diff
@@ -25,31 +27,35 @@
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 """
 
-import os, string, glob, copy, sys, shutil
-
-from docutils import nodes
-from docutils.parsers.rst import directives, Directive
+import os, string, glob, copy, re, sys, shutil
 
 import nbformat
 
-from nbconvert import NotebookExporter, PythonExporter, HTMLExporter
-from nbconvert.preprocessors import (ExecutePreprocessor, CellExecutionError,
-                                     Preprocessor)
+from docutils.parsers.rst import directives, Directive
+from docutils.statemachine import string2lines
+from docutils.utils import new_document
+from myst_nb.parser import nb_to_tokens, nb_output_to_disc, tokens_to_docutils
```

Comment:
Looks like these utilities are doing the lifting from `myst_nb` in a way that integrates fairly well with the old code. Seems like a good approach.

Reply:
Yeah it could be worse. Sphinx documentation is terrible as always so it took me ages to understand how to render docutils nodes, which I could then return in the directive.